### PR TITLE
WIP Create 'wp-comments' core package

### PR DESCRIPTION
**Pull request by DAreRodz**
Closes https://github.com/frontity/saturn-theme/issues/195

### CODE REVIEW

File: core/packages/wp-comments/pwa/stores/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      const {
+        _embedded: { replies },
+      } = getRoot(self).connection.entity(type, id).raw;
+      return replies ? replies[0].map(normalizeComment) : [];
+    },
+  }))
+  .actions(self => ({
+    create: flow(function* create({ id, name, email, url, content }) {
+      const { api } = getEnv(self).connection.WpApi;
+      // CODE BELOW IS NOT WORKING
```
Comment (luisherranz) [inline core/packages/wp-comments/pwa/stores/index.js:38]: Why is this not working?

File: core/server/index.js [hunk 2]
```diff
@@ -29,7 +29,10 @@ const customCssModule = require(`../packages/custom-css/${
 const oneSignalModule = require(`../packages/one-signal/${
   process.env.MODE
 }/server`);
-const disqusCommentsModule = require(`../packages/disqus-comments/${
+// const disqusCommentsModule = require(`../packages/disqus-comments/${
```
Comment (luisherranz) [inline core/server/index.js:32]: Why is disqus commented out? How a user can choose what to use?

File: core/client/index.js [hunk 3]
```diff
@@ -22,7 +22,10 @@ const customCssModule = require(`../packages/custom-css/${
 const oneSignalModule = require(`../packages/one-signal/${
   process.env.MODE
 }/client`);
-const disqusCommentsModule = require(`../packages/disqus-comments/${
+// const disqusCommentsModule = require(`../packages/disqus-comments/${
```
Comment (luisherranz) [inline core/client/index.js:25]: Why is disqus commented out? How a user can choose what to use?

**Review by luisherranz — CHANGES_REQUESTED** [review]
@DAreRodz is this PR finished? I can't figure out how you thought a user could choose between disqus and wp comments and there's a comment that indicates that creating comments don't work.

**Comment by DAreRodz** [comment]
No, @luisherranz, this PR is not finished yet, I forgot to include WIP in the PR title. Fixed now.

**Comment by luisherranz** [comment]
Ok.

**Comment by luisherranz** [comment]
Are you going to create two separate packages for disqus and wp comments?

**Comment by DAreRodz** [comment]
Yes, that's what I planned to do, and so you could choose the one you want.

**Comment by luisherranz** [comment]
Perfect. Thanks @DAreRodz!

**Pull request closed without merging**